assets/app/features/upgrade/controllers/upgrade7-openstack-services.controller.js

```diff
@@ -48,19 +51,16 @@
                     
                 ).finally(function() {
                     // Either on sucess or failure, the openStackServices has been completed.
-                    vm.openStackServices.completed = true;
-
-                    // Update openStackServices validity
-                    
-
+                    vm.openStackServices.completed = true;              
+                    vm.openStackServices.running = false;
```

Comment:
This will hide the spinner before the backup operation is completed. Note that backup creation is an async operation which returns immediately.